### PR TITLE
Pyxis 1.8.13

### PR DESCRIPTION
**Pull request by kioan000**
### 1.8.13
**sass changes**

Accordion:
- Single accordion don't have visible border bottom when closed

Accordion Group:
- Accordion in accordion group have separation borders (all except last)

Button:
- Adjacent buttons doesn't have default margin anymore, use button group instead

Button Group:
- Buttons in button group have margin
- Removed duplicated flex justify modificators, use utility class instead
- New specific `m-btnGroup--coverFluid` modificator that allow inner buttons to cover it fully and as much as possible equally growing. Margins are supported until 4 buttons

Utils:
- New `withNElements(selector, n)` mixin that given a selector and a number generates a class that applies only if there are N child elements with that selector
- Added `space-evenly` as justify helper

**sass fix**

Button:
- `a-btn--primary` now have the same height of other buttons (invisible border)
- fixed margin in case of icon due to a duplicated margin setting
- `a-btn--smallOnBreakpoint[breakpoint]` now is applied correctly even on 0 to [small] media query

Links
- `a-link` now have line-height and font-size explicitly setted 
- `a-link--standalone` correct line-height
- `a-link--standalone` parametric without transform usage

**project specific**
- Updated default transpiler dependencies

### CODE REVIEW

**Comment by kioan000** [comment]
- [x] `m-accordionGroup`
- [x] `m-btnGroup` e btn liquidi
- [x] correzione della classe `.a-btn--smallOnBreakpointXSmall` (attualmente ha una mq min-width e una max-width, dovrebbe avere solo la max-width)
- [x] correzione del link `a-link--standalone` con freccia che si sovrappone al testo in caso di testo molto lungo
- [x] correzione del link `a-link--standalone` con problema di allineamento verticale delle freccia

File: src/scss/02_atoms/_accordion.scss [hunk 1]
```diff
@@ -61,13 +60,17 @@
   text-decoration: none;
   width: 100%;
 
-  .m-group--dark & {
+  .a-accordion.is-open & {
+    border-bottom: 1px solid color(shape);
+  }
```
Comment (alessiosantangelo) [inline src/scss/02_atoms/_accordion.scss:61]: Spazio

File: src/scss/02_atoms/_accordion.scss [hunk 2]
```diff
@@ -61,13 +60,17 @@
...
+  .a-accordion.a-accordion--dark.is-open {
     border-bottom: 1px solid color(shape, dark);
   }
 
-  .a-accordion--dark & {
+  .m-group--dark & {
```
Comment (alessiosantangelo) [inline src/scss/02_atoms/_accordion.scss]: Cancella la classe `m-group` e tutti i suoi riferimenti. Credo sia un refuso.

File: src/scss/03_molecules/_accordionGroup.scss [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
-  justify-content: flex-start;
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  
-  &.m-accordion-group--dark {
-    background: color(backgroundAlt, base);
-    color: color(textAlt, light);
+.m-accordionGroup {
+  .a-accordion:not(:last-child) {
```
Comment (alessiosantangelo) [inline src/scss/03_molecules/_accordionGroup.scss]: `.m-accordionGroup .a-accordion:not(:last-child) .a-accordion__toggle {`

`.m-accordionGroup .a-accordion.a-accordion--dark:not(:last-child) .a-accordion__toggle {`

File: src/scss/03_molecules/_buttonGroup.scss [hunk 4]
```diff
@@ -1,13 +1,249 @@
...
+  flex-flow: row wrap;
   padding: 15px 0;
+
+  .a-btn {
+    margin: $btnTopMargin $btnSideMargin;
+  }
 }
 
+
 .m-btnGroup.m-btnGroup--left {
```
Comment (alessiosantangelo) [inline src/scss/03_molecules/_buttonGroup.scss]: Questi si possono rimuovere.

File: src/scss/03_molecules/_buttonGroup.scss [hunk 5]
```diff
@@ -1,13 +1,249 @@
...
+.m-btnGroup.m-btnGroup--space-between {
+  justify-content: space-between;
+}
+
+/**
+    100% fluid group with buttons that automatically grow that keeps gutters between buttons
+*/
+
+.m-btnGroup.m-btnGroup--coverFluid {
+  .a-btn {
```
Comment (alessiosantangelo) [inline src/scss/03_molecules/_buttonGroup.scss]: `.m-btnGroup.m-btnGroup--coverFluid .a-btn`

File: src/scss/03_molecules/_buttonGroup.scss [hunk 6]
```diff
@@ -1,13 +1,249 @@
...
+  /**
+
+    Automargin magic
+
+  */
+
+  @mixin aloneInRowMargins {
+    margin-left: 0;
+    margin-right: 0;
+  }
```
Comment (alessiosantangelo) [inline src/scss/03_molecules/_buttonGroup.scss:40]: A capo

File: src/scss/03_molecules/_buttonGroup.scss [hunk 7]
```diff
@@ -1,13 +1,249 @@
...
+      &:nth-child(3) {
+        @include lastInRowMargins;
+      }
+    }
+
+  }
+
+  /* When four buttons small */
+  @include withNElements('.a-btn.a-btn--small', 4) {
+    &:nth-child(1), &:nth-child(3) {
```
Comment (alessiosantangelo) [inline src/scss/03_molecules/_buttonGroup.scss]: Manda sempre a capo dopo `,`

**Review by alessiosantangelo — CHANGES_REQUESTED** [review]
Due cavolatine da eliminare e un paio di refusi

**Review by alessiosantangelo — APPROVED** [review]

**Pull request merged**